docs/releasehistory.rst

```diff
@@ -22,6 +22,18 @@ API-breaking changes
   there are no special behaviors that are accessed in the case of partially-licensed OpenEye backends. The
   new behavior of this method is the same as if the default value above is always provided.
 
+Behavior Changed
+""""""""""""""""
+- `PR #583 <https://github.com/openforcefield/openforcefield/pull/583>`_: Methods
+  such as :py:meth:`Molecule.from_rdkit <openforcefield.topology.Molecule.from_rdkit>`
+  and :py:meth:`Molecule.from_openeye <openforcefield.topology.Molecule.from_openeye>`,
+  which delegate their internal logic to :py:class:`ToolkitRegistry <openforcefield.utils.ToolkitRegistry>`
+  functions, now guarantee that they will return an object of the correct type. Previously,
```

Comment:
nit: my first reading of this make me wonder if the changed behavior was applicable to `Molecule` itself, not only its subclasses
```suggestion
  functions, now guarantee that they will return an object of the correct type when being called on ``Molecule``-derived classes. Previously,
```